docs/runtimes/mlrun_code_annotations.ipynb

```diff
@@ -0,0 +1,1253 @@
+{
```

Comment:
<p><code># mlrun: start-code</code> and <code># mlrun: end-code</code> annotations are relative to their positions inside the code block. Notice how the assignments to <code>function_name</code> below <code># mlrun: end-code</code> don't override the assignment between the annotations in the function's context.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/1456/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>